consensus/state_processing/src/common/get_attesting_indices.rs

```diff
@@ -138,8 +142,11 @@ pub mod attesting_indices_electra {
             } else {
                 return Err(Error::NoCommitteeFound);
             }
+        }
 
-            // TODO(electra) what should we do when theres no committee found for a given index?
```

Comment:
Also i removed this todo because it looks like we error in this case already (L139) which looks right to me